src/components/mapPageComponents/ducks/types.ts

```diff
@@ -70,6 +70,8 @@ interface SiteFeaturePropertiesResponse {
   address: string;
   lat: number;
   lng: number;
+  plantingDate?: number;
+  adopterId?: string;
```

Comment:
❓ Do these have to be optional? Some of these other properties might also be null (ex. `address`), so should those be optional too? Either way, seems best to keep it consistent

Reply:
Yes, I believe these have to be optional because the api may return null for these values. Good catch on the address, that should be optional as well